### DISCUSSION TITLE
Can we force a link to open in a popup inside the Detailed List layout?

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#3980] [Q&A]
I have the following setting inside my PnP Modern search result:-

![Save1](https://github.com/user-attachments/assets/ea76b9be-1e80-4738-8312-1dc2e44866c7)

![Save2](https://github.com/user-attachments/assets/06324e7e-c678-4dfa-9750-432fdcbfde53)

![save3](https://github.com/user-attachments/assets/8e121d2e-d80d-4add-bbac-4e68140d6be3)

![save4](https://github.com/user-attachments/assets/30357ca9-9cc6-427a-9902-d62c2727fa4e)

Here is the code:-

```
<a href="{{slot item @root.slots.PreviewUrl}}" data-interception="off" target="_blank" style="color: {{@root.theme.semanticColors.link}}">
	{{slot item @root.slots.Destination}}
</a>
```

Currently the link will open in a new tab. so can i force the link to open in  a popup window ? instead of a new tab?

Thanks

**Comment by wobba** [comment]
There is no HTML which will open a modal/pop-up window. To do that you need a custom web component.
